### ISSUE TITLE
DBeaver may ignore configured key commands

### ISSUE THREAD

**Issue by esauloff** [dbeaver/dbeaver#20612]
### Description

DBeaver may ignore configured key commands sometimes.
Issue observed on MacBook Pro M1 with macOS Ventura, when external keyboard is connected.

### DBeaver Version

Community Edition v23.1.2.202307091549

### Operating System

macOS Ventura v13.2.1 (22D68)

### Database and driver

1. Database: PostgreSQL 11.3
2. Driver: PostgreSQL JDBC Driver 42.5.2

### Steps to reproduce

**Prerequisites**
Connect any regular US ANSI 104 keys standard layout keyboard.

**Steps**
1. Configure user key commands: go to DBeaver menu --> Settings --> User Interface --> Keys.
2. Define `Line Start` command as `Home` button:
Binding column shows single `↖` symbol, `When` is `Editing Text`, and `User` column shows `U` symbol.
3. Define `Text Start` command as `Command+Home` buttons:
Binding columns shows `⌘↖` combination, `When` is `Editing Text` and `User` column shows `U` symbol.
4. Apply and Close Preferences window.
5. In script editor, right away keys work as expected, `Home` brings cursor to line start and with `Command+Home` cursor goes to entire text start.
6. Press PgUp or PgDown buttons.
7. After step 6, user defined keys for line start and text start are ignored.
8. Restart application and keys work as expected again, until PgUp or PgDown is pressed.

**Expected**
User defined keys work as expected, regardless of other keys.

**Key Events**
Captured with Karabiner Event Viewer application.
* `Home` button
```
{
  "name": {"key_code":"home"},
  "usagePage": "7 (0x0007)",
  "usage": "74 (0x004a)",
  "misc": ""
}
```
* `Command` button
```
{
  "name": {"key_code":"left_option"},
  "usagePage": "7 (0x0007)",
  "usage": "226 (0x00e2)",
  "misc": "flags left_option"
}
```
* `PgUp` button
```
{
  "name": {"key_code":"page_up"},
  "usagePage": "7 (0x0007)",
  "usage": "75 (0x004b)",
  "misc": ""
}
```
* `PgDown` button
```
{
  "name": {"key_code":"page_down"},
  "usagePage": "7 (0x0007)",
  "usage": "78 (0x004e)",
  "misc": ""
}
```

**Comment by E1izabeth** [comment]
Sounds like a duplicate of #5931 or/and #9414

**Comment by Destrolaric** [comment]
Duplicate of https://github.com/dbeaver/dbeaver/issues/9414
